src/pages/Swap/index.tsx

```diff
@@ -193,8 +193,8 @@ export function Swap({
   const trace = useTrace()
 
   // token warning stuff
-  const prefilledInputCurrency = useCurrency(initialInputCurrencyId)
-  const prefilledOutputCurrency = useCurrency(initialOutputCurrencyId)
+  const prefilledInputCurrency = useCurrency(initialInputCurrencyId, chainId)
+  const prefilledOutputCurrency = useCurrency(initialOutputCurrencyId, chainId)
```

Comment:
"In most cases, chainId should refer to the connected chain, i.e. `useWeb3React().chainId`. However if this component is being used in a context that displays information from a different, unconnected chain (e.g. the TDP), then chainId should refer to the unconnected chain."